src/AngleSharp.Diffing.Tests/Strategies/AttributeStrategies/OrderedStyleAttributeComparerTest.cs

```diff
@@ -0,0 +1,58 @@
+﻿using AngleSharp.Diffing.Core;
+using Shouldly;
+using Xunit;
+
+namespace AngleSharp.Diffing.Strategies.AttributeStrategies
+{
+    public class OrderedStyleAttributeComparerTest : DiffingTestBase
+    {
+        public OrderedStyleAttributeComparerTest(DiffingTestFixture fixture) : base(fixture)
+        {
+        }
+
+        [Fact(DisplayName = "When attribute is not style the current decision is used")]
+        public void Test001()
+        {
+            var comparison = ToAttributeComparison(@"<p foo=""bar"">", "foo", @"<p foo=""zab"">", "foo");
+            StyleAttributeComparer.Compare(comparison, CompareResult.Different).ShouldBe(CompareResult.Different);
+            StyleAttributeComparer.Compare(comparison, CompareResult.Same).ShouldBe(CompareResult.Same);
+            StyleAttributeComparer.Compare(comparison, CompareResult.Skip).ShouldBe(CompareResult.Skip);
+        }
+
+        [Theory(DisplayName = "When style attributes has different values then Different is returned")]
+        [InlineData(@"<p style=""color: red"">", @"<p style=""color: black"">")]
+        [InlineData(@"<p style=""color: red"">", @"<p style=""text-align:center"">")]
+        [InlineData(@"<p style=""color: red"">", @"<p style=""color: red;text-align:center"">")]
+        [InlineData(@"<p style=""color: red;text-align:center"">", @"<p style=""color: red"">")]
+        public void Test002(string control, string test)
+        {
+            var comparison = ToAttributeComparison(control, "style", test, "style");
+            OrderedStyleAttributeComparer.Compare(comparison, CompareResult.Unknown).ShouldBe(CompareResult.Different);
+        }
+
+        [Fact(DisplayName = "Comparer should correctly ignore insignificant whitespace")]
```

Comment:
Does this also work with style infos in different order?
I think  yes, because AngleSharp is removing the insignificant whitespace automatically for us.

Reply:
Good question, will test that.